MiniApp/Classes/DownloadOperation.swift

```diff
@@ -0,0 +1,70 @@
+class DownloadOperation {
```

Comment:
Looks like there aren't any unit tests for this class?

Reply:
Working on it